theme/index.hbs

```diff
@@ -104,11 +104,15 @@
             <h1 class="menu-title">{{ book_title }}</h1>
 
             <div class="right-buttons">
-                {{#if git_repository_url}}
-                    <a href="{{git_repository_url}}" title="Git repository" aria-label="Git repository">
-                        <i id="git-repository-button" class="fa {{git_repository_icon}}"></i>
-                    </a>
-                {{/if}}
+                <a href="https://alan-lang.org" title="Home">
+                    <i>Home</i>
+                </a>
+                <a href="https://alan-lang.org/blog.html" title="Blog">
```

Comment:
Nit: I saw this before. If we restructure the blog so it's under `/blog/index.html` we can make this link a prettier-looking `https://alan-lang.org/blog` instead.

Reply:
Not really on this PR, but just sayin'.

Reply:
I agree this looks ugly, but unfortunately that would redirect to `https://docs.alan-lang.org/blog` and we want `https://alan-lang.org/blog`

Reply:
wait, what? How? I thought the blog was on the homepage, not the docs?

Reply:
But this is the docs repo? I don't follow either. If this is the docs repo then `/blog.html` will take you to `https://docs.alan-lang.org/blog.html`, but that doesn't exist

Reply:
Right, that's why I said it's not on this PR, but that the homepage repo structure probably ought to be redone as `/blog/index.html` instead of `/blog.html` and *then* this is updated a second time with the better URL.

Reply:
oh I see what you mean. The url is set by mdbook, but there might be a way to change it